### DISCUSSION TITLE
ssl-root-cas package not working

### DISCUSSION THREAD

**Discussion by abhyudayasrinet** [vercel/next.js#21928] [Help]
**What version of Next.js are you using?**

10.0.5

**What version of Node.js are you using?**

12.13

**What browser are you using?**

Edge, Chrome, Firefox

**What operating system are you using?**

Windows, macOS

**How are you deploying your application?**

Vercel

**Describe the Bug**

I have my website `domain.com` hosted on Vercel. The next.js application talks to a Laravel API deployed at a `subdomain.domain.com` on AWS for server-side rendering.
I bought a separate SSL certificate for the wildcard domains and added the CAA entries to the DNS for the CA authorities. 
I see the certificate verified and working fine in the browsers. However, the server-side rendering requests are failing with the following error
```
event - build page: /user/[profile_id]
wait  - compiling...
event - compiled successfully
Error: unable to verify the first certificate
    at TLSSocket.onConnectSecure (_tls_wrap.js:1321:34)
    at TLSSocket.emit (events.js:210:5)
    at TLSSocket._finishInit (_tls_wrap.js:794:8)
    at TLSWrap.ssl.onhandshakedone (_tls_wrap.js:608:12) {
  code: 'UNABLE_TO_VERIFY_LEAF_SIGNATURE',
  config: {
    url: 'https://subdomain.domain.com/api/account/john-doe',
    method: 'get',
    headers: {
      Accept: 'application/json',
      Authorization: '',
      token: '',
      'User-Agent': 'axios/0.21.1'
    },
    .
    .
```

I came across this package [ssl-root-cas](https://www.npmjs.com/package/ssl-root-cas), and the issue is fixed in my local development environment and the pages load fine. I added this snippet to `next.config.js`

```
'use strict';
var rootCas = require('ssl-root-cas').create();
rootCas.addFile(__dirname + '/domain.ca-bundle');
// default for all https requests
// (whether using https directly, request, or another module)
require('https').globalAgent.options.ca = rootCas;
```

However, this doesn't seem to be working when I deploy to my staging site on Vercel.

**Expected Behavior**

The server-side rendering calls should be able to send requests and load data from `subdomain.domain.com` normally

**To Reproduce**

NA

**Comment by fernandotresa** [comment]
Same error here my friend, would you be able to solve it?
